README.md

```diff
@@ -97,6 +97,10 @@ Have you ever been at "remote" party or tried to listen few parallel discussions
 
 - [DDD by Example, Factory](https://github.com/ddd-by-examples/factory)
 
+## Tools
+
+- [Realtimeboard](https://realtimeboard.com/)
```

Comment:
RealtimeBoard is now called [Miro](https://miro.com/blog/features/realtimeboard-is-now-miro/) 😉

Reply:
By the time this pull request is merged, it will be rebranded again :D